data/human/news.txt

```diff
@@ -1871,8 +2058,11 @@ news "syndicate pa system"
 			"Dear spaceship captains: please be aware that our outfitters are open all hours of the day. There is never a reason to delay the purchase of your next piece of Syndicate equipment."
 			"Dear traveler: we recommend you to have dinner in our local ${culinary venue} '${restaurant names}'. You will not be disappointed."
 			"Attention course participants: The course 'Efficient Passenger Logistics' has been canceled because the trainer got stuck in transit."
+			"Attention course participants: The course 'Communication for Professionals' is canceled because the provider miscommunicated the date to the instructor."
```

Comment:
```suggestion
			"Attention course participants: The course 'Communication for Professionals' is canceled because the provider failed to provide the date to the instructor."
```